### ISSUE TITLE
Make sure *all* EDLogs 'self' members are initialised

### ISSUE THREAD

**Issue by Athanasius** [EDCD/EDMarketConnector#996]
```python
2021-04-10 23:49:05.193 - DEBUG - 154716:153680:153680 monitor.EDLogs.worker:321: Now at end of latest file.
Exception in thread Journal worker:
Traceback (most recent call last):
  File "threading.pyc", line 954, in _bootstrap_inner
  File "threading.pyc", line 892, in run
  File "monitor.pyc", line 334, in worker
AttributeError: 'EDLogs' object has no attribute 'systempopulation'
```

EDLogs.systempopulation is only set in one place and never properly initialised.

**Comment by Athanasius** [comment]
It turns out this was literally only self.systempopulation

**Comment by Athanasius** [comment]
Closed by #997
